doc_admin/AdminManual.adoc

```diff
@@ -650,55 +658,75 @@ Default configuration is available in the _settings.properties.dist_ file:
 
 [source,properties]
 ----
+###########################################
 # This file contains Open Hospital settings
+###########################################
+# external settings
+MODE=OH_MODE
+###########################################
+# internal settings, modules, directories
 LANGUAGE=OH_LANGUAGE
-SINGLEUSER=yes
+SINGLEUSER=YES_OR_NO
+DEBUG=no
+DOC_DIR=OH_DOC_DIR
+PATIENTPHOTOSTORAGE=PHOTO_DIR
+INTERNALVIEWER=yes
+SMSENABLED=no
+VIDEOMODULEENABLED=yes
+XMPPMODULEENABLED=no
+###########################################
+# application settings
+ENHANCEDSEARCH=no
+INTERNALPHARMACIES=yes
+LABEXTENDED=yes
+LABMULTIPLEINSERT=yes
+MATERNITYRESTARTINJUNE=no
+MERGEFUNCTION=yes
+OPDEXTENDED=yes
+PATIENTEXTENDED=yes
+PATIENTVACCINEEXTENDED=yes
+###########################################
```

Comment:
I would replace ################## with just empty lines

Reply:
I think it's clearer in this way, we are using this standard also in makefile and scripts

Reply:
makefile and scripts are not meant to be opened by users

parts like these ones result very confused and packed (like double title)
```
###########################################
# This file contains Open Hospital settings
###########################################
# external settings
MODE=OH_MODE
###########################################
# internal settings, modules, directories
LANGUAGE=OH_LANGUAGE
SINGLEUSER=yes
SINGLEUSER=YES_OR_NO
DEBUG=no
```

much more "bright" this way instead (less is more, spare characters, save the planet :) )

```
# This file contains Open Hospital settings

# external settings
MODE=OH_MODE

# internal settings, modules, directories
LANGUAGE=OH_LANGUAGE
SINGLEUSER=yes
SINGLEUSER=YES_OR_NO
DEBUG=no
```